docs/18-guides/02-security/index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: App Security Guide
+description: Secure your Nordcraft app by avoiding frontend secrets and using a backend to handle sensitive API operations safely
```

Comment:
```suggestion
description: Learn how to keep your Nordcraft front-end app secure by being mindful of exposing secret API credentials and other things to avoid.
```